src/snmalloc/global/globalalloc.h

```diff
@@ -4,7 +4,7 @@
 
 namespace snmalloc
 {
-  template<SNMALLOC_CONCEPT(IsConfig) Config>
+  template<SNMALLOC_CONCEPT(IsConfig) Config = Config>
```

Comment:
1. I'm a little surprised these aren't triggering shadowing warnings?
2. Are these functions ever invoked with non-default `Config`s?  If not, maybe convert the template parameter and concept annotation to a suitable `static_assert` `requires`?

Reply:
I did `Config_` in #750.